Qcom/Platforms/Msm8960/Library/QcomPlatformUartDmLib/QcomPlatformUartDmLib.c

```diff
@@ -28,11 +28,15 @@ VOID LibQcomPlatformUartDmGpioConfig(UINT8 id)
 
 		case GSBI_ID_5:
 			/* configure rx gpio */
-			gGpioTlmm->gpio_tlmm_config(52, 2, GPIO_INPUT, GPIO_NO_PULL,
-							 GPIO_8MA, GPIO_DISABLE);
+			gGpioTlmm->SetFunction(52, 2);
+			gGpioTlmm->SetDriveStrength(52, 8);
+			gGpioTlmm->SetPull(52, GPIO_PULL_NONE);
+			gGpioTlmm->DirectionInput(52);
 			/* configure tx gpio */
```

Comment:
pls add an empty line between the rx and tx blocks like in device/xiaomi/aries.
Same for all the other GSBIs